src/sensor.cpp

```diff
@@ -9,7 +10,19 @@ void Sensor::setPin(int set)
 
 int Sensor::getValue()
 {
-    if (analogRead(pin) > 1000)
+    time_t endTime;
+    int i = 0;
+    int totalValue = 0;
+    int averageValue = 0;
+    endTime = time(NULL) + 30;
```

Comment:
```suggestion
    endTime = time(NULL) + measureTime;
```
変数で変更できるようにしてもらえるとわかりやすいかと